src/pages/api/shared.ts

```diff
@@ -235,3 +235,53 @@ export async function getCurrentTetuVeBALGaugeVotes() {
 
 	return percents
 }
+
+// Fetch gauge current bias & handle Quest blacklist feature
+// Remove blacklisted addresses votes from gauge bias
+async function getGaugeBiasBlacklist(gauge: string, blacklist: string[]): Promise<BigNumber> {
```

Comment:
Note to self: will need to hide warden on previous rounds, since we are only retrieving current data.